src/index.html

```diff
@@ -7,15 +7,12 @@
     <link rel="stylesheet" href="https://js.arcgis.com/4.10/esri/css/main.css">
     <link rel="stylesheet" href="index.css">
     <script>
-      // Current path minus everything after the last forward slash
-      var locationPath = location.pathname.replace(/\/[^\/]+$/, "");
```

Comment:
So you don't have to do this because you moved the assets to `src`?

Reply:
It was an intermediate variable whose use was removed below.

Reply:
And also, I was wrong.